newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -165,8 +174,18 @@ export const GameDetailsDialog = ({
         orientation: project.getOrientation(),
         // The thumbnailUrl is updated only when a build is made public.
       });
-      const authorAcls = getAclsFromAuthorIds(project.getAuthorIds());
-      await setGameUserAcls(getAuthorizationHeader, id, gameId, authorAcls);
+      try {
+        const authorAcls = getAclsFromUserIds(
+          project.getAuthorIds().toJSArray()
+        );
+        const ownerAcls = getAclsFromUserIds(partialGameChange.ownerIds);
+        await setGameUserAcls(getAuthorizationHeader, id, gameId, {
+          ownership: ownerAcls,
+          author: authorAcls,
+        });
+      } catch (error) {
+        console.error('Unable to update the game owners or authors:', error);
+      }
```

Comment:
When this catch happen, the game detail dialog will show a loading indicator indefinitely because `publicGame`is null and `onGameUpdated` is never called to refetch it.
This is not a big issue because it will probably only happen if the service can't be reached, but I find it odd that the component force a refetch with knowledge of other component behavior.

Reply:
I think you mean the catch just below?
In this case, can we, in the catch, call again `setPublicGame(oldGame)` (you defined the variable but haven't used it :)
And also show an error box with `showErrorBox()` asking to try again? 

As you mention, it shouldn't happen much, so no need for a fancy UI